### PR TITLE
Modulo4 matias fino

### PR DESCRIPTION
**Pull request by MatiasFino**
Clean architecture applied to my project. Also lint added
image
image

### CODE REVIEW

**Review by MarcosCGlobant — APPROVED** [review]

**Review by FranciscoGaraventa — APPROVED** [review]
LG

File: lib/data/repository/genre_from_json.dart [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import 'dart:convert';
 import 'package:flutter/services.dart';
 
-import '../i_genre_repository.dart';
+import '../../domain/repository/i_genre_repository.dart';
 
 class GenresFromJson extends GenresRepository {
```
Comment (matiasilveiro-gl) [inline lib/data/repository/genre_from_json.dart]: This should be `implement` instead.

File: lib/data/repository/movie_from_json.dart [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import 'dart:convert';
 import 'package:flutter/services.dart';
 
-import '../i_movie_repository.dart';
-import '../../../data_classes/movie.dart';
+import '../../domain/repository/i_movie_repository.dart';
+import '../../domain/entity/movie.dart';
 
 class MoviesFromJson extends MovieRepository {
```
Comment (matiasilveiro-gl) [inline lib/data/repository/movie_from_json.dart]: This should be `implement` instead.

**Review by matiasilveiro-gl — APPROVED** [review]
LGTM! Minor comments

File: lib/data/repository/genre_from_json.dart (hunk 1 above)
Comment (MatiasFino) [reply lib/data/repository/genre_from_json.dart]: done

File: lib/data/repository/movie_from_json.dart (hunk 2 above)
Comment (MatiasFino) [reply lib/data/repository/movie_from_json.dart]: solved

**Pull request merged**